semcore/data-table/src/types.ts

```diff
@@ -22,7 +22,8 @@ export type Column<
   resizable?: boolean;
   sortable?: boolean | SortDirection;
   sortDirection: SortDirection;
-  cssVar: string | string[];
+  varWidth: string;
+  setVar: boolean;
```

Comment:
Do we really need to expose so low-level apis?

Reply:
This is an internal api 
https://github.com/semrush/intergalactic/blob/9b89f4c926c1fe14021f3a7d5f19041a493c7dc0/semcore/data-table/src/DataTable.tsx#L157

Reply:
As I see from [coming examples](https://github.com/semrush/intergalactic/pull/415/files/9b89f4c926c1fe14021f3a7d5f19041a493c7dc0#diff-b80726e2311033aa13b20cf627bf4fcf036a4d883e9211cbe459d6e60a9c4885R38-R41) these apis are intended to be available for users so it's kinda enough to perceive them as public ones

Reply:
These are still internal types. And I just did not make public ones. Now I have [added](https://github.com/semrush/intergalactic/pull/415/files/9b89f4c926c1fe14021f3a7d5f19041a493c7dc0#diff-0fae13344b5ab0b2735577659c2fdeac367710308c912123bb21068a1cb63d2aR102) them.